openshift/templates/cakephp-mysql.json

```diff
@@ -354,6 +354,12 @@
   ],
   "parameters": [
     {
+      "name": "NAME",
+      "displayName": "Name",
+      "description": "The name assigned to all of the objects defined in this template.",
```

Comment:
should still say frontend objects (or some other wording) because the DB objects still get named via a different parameter (the DATABASE_SERVICE_NAME parameter)